imports/plugins/core/accounts/client/components/addressBookGrid.js

```diff
@@ -0,0 +1,224 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Components, registerComponent } from "@reactioncommerce/reaction-components";
+
+class AddressBookGrid extends Component {
+  static propTypes = {
+    /**
+     * array of address objects
+     */
+    addressBook: PropTypes.arrayOf(PropTypes.shape({
+      _id: PropTypes.String,
+      fullName: PropTypes.String,
+      address1: PropTypes.String,
+      addresss2: PropTypes.String,
+      postal: PropTypes.String,
+      city: PropTypes.String,
+      region: PropTypes.String,
+      country: PropTypes.String,
+      phone: PropTypes.String,
+      isBillingDefault: PropTypes.Bool,
+      isShippingDefault: PropTypes.Bool,
+      isCommercal: PropTypes.Bool
+    })),
+    /**
+     * selects an address to be edited and renders the AddressBookForm
+     */
+    edit: PropTypes.func,
+    /**
+     * removes the selected address
+     */
+    remove: PropTypes.func,
+    /**
+     * selects a default shipping or billing address
+     */
+    select: PropTypes.func
+  }
```

Comment:
Add default no-ops for all functions since you call them without checking whether they're set.

```js
static defaultProps = {
  edit() {},
  remove() {},
  select() {}
};
```